process.c

```diff
@@ -8570,19 +8570,87 @@ proc_warmup(VALUE _)
 /*
  * Document-module: Process
  *
- *   The module contains several groups of functionality for handling OS processes:
- *
- *   * Low-level property introspection and management of the current process, like
- *     Process.argv0, Process.pid;
- *   * Low-level introspection of other processes, like Process.getpgid, Process.getpriority;
- *   * Management of the current process: Process.abort, Process.exit, Process.daemon, etc.
- *     (for convenience, most of those are also available as global functions
- *     and module functions of Kernel);
- *   * Creation and management of child processes: Process.fork, Process.spawn, and
- *     related methods;
- *   * Management of low-level system clock: Process.times and Process.clock_gettime,
- *     which could be important for proper benchmarking and other elapsed
- *     time measurement tasks.
+ * \Module \Process represents a process in the underlying operating system.
```

Comment:
To escape `Module` is that it is a common word, but `Process` is not a common word here.
I think it is better to mark up as code.
```suggestion
 * \Module +Process+ represents a process in the underlying operating system.
```

Reply:
@nobu, we have hundreds (if not thousands) of instances like \Process.  If ok with you, I'd like to defer this and propose a usage in the documentation guide.